### PR TITLE
Total population

### PR DESCRIPTION
**Pull request by longhotsummer**
1. Simple table for total population
2. When searching for tables, sort results alphabetically.

@xybrnet

### CODE REVIEW

File: censusreporter/apps/census/static/js/comparisons.js [hunk 1]
```diff
@@ -990,6 +990,9 @@ function Comparison(options) {
 
             return Bloodhound.tokenizers.whitespace(fields.join(' '));
         },
+        sorter: function(a, b) {
+            return a.table_id.localeCompare(b.table_id);
```
Comment (xybrnet) [inline censusreporter/apps/census/static/js/comparisons.js]: I'm not sure what this does?

Comment (longhotsummer) [inline censusreporter/apps/census/static/js/comparisons.js]: It tells the Bloodhound instance how to sort search results. Given two results a and b, return an integer telling it how the two compare. In this case, we want to compare the table ids alphabetically, which are the full names of the tables.

See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort

Comment (xybrnet) [inline censusreporter/apps/census/static/js/comparisons.js]: Got it, thanks.

**Comment by xybrnet** [comment]
:+1:

**Pull request merged**